packages/edit-post/src/editor.js

```diff
@@ -87,7 +85,7 @@ function Editor( {
 			template:
 				isFSETheme &&
 				postObject &&
-				! __unstableIsAutodraftPost() &&
+				getCurrentPost().status !== 'auto-draft' &&
```

Comment:
Why don't we need the check `isSavingPost` here?

Reply:
the isSaving  was only needed previously because of the optimistic update. It meant that even if a post was  "auto-draft", when the save starts, the status changed to "publish" or "draft" so we had to ensure the saving was not in progress  to check properly for "auto-draft" but with the change in this PR, there's no optimistic update which means the post-status is only updated after the  post is actually saved (responsive received from the server).

The new behavior is simpler and also more stable. In the previous version, the ` __unstableIsAutodraftPost` call would have returned "true" even for a published post that was being updated.

Reply:
Thanks for explaining!